presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/CachingHiveMetastore.java

```diff
@@ -84,6 +84,8 @@
         ALL, PARTITION
     }
 
+    public static final int PARTITION_CACHE_MAX_COLUMN_COUNT = 100;
```

Comment:
what's going on with this? This seems like a dummy value for when the max column count isn't actually going to get used (otherwise we should use the actually configured value).  I would inline 0 in all those places instead.

Reply:
Agree. Instead of 0, I set it to a high value 10_000 so that invalidating partition cache based on column count will NOT kick in for the tests.